src/quick-tour/increase-average-order-value.md

```diff
@@ -4,28 +4,28 @@ title: Increase Average Order Value
 
-[Coupons]({{ site.baseurl }}{% link marketing/price-rules-cart-coupon.md %})
-:  Create limited-time offers and coupons that customers can scan with their phone and apply to a purchase.
-
-[Product Suggestions]({{ site.baseurl }}{% link marketing/price-rules-cart-coupon.md %})
-:  Another way to increase AOV is to offer suggestions for related products and opportunities to up-sell and cross-sell at strategic points along the path to conversion.
+Coupons
+:  Create limited-time offers and [coupons]({{ site.baseurl }}{% link marketing/price-rules-cart-coupon.md %}) that customers can scan with their phone and apply to a purchase.
 
+Product Suggestions
+:  Another way to increase AOV is to offer [suggestions]({{ site.baseurl }}{% link catalog/related-products-up-sells-cross-sells.md %}) for related products and opportunities to up-sell and cross-sell at strategic points along the path to conversion.
 <!--{% if "Default.EE-B2B" contains site.edition %}-->
-[Email Reminders]({{ site.baseurl }}{% link marketing/email-reminder-rules.md %})
-:  Send automated reminder emails to customers who have added items to their carts or wish lists, but haven’t made a purchase. A variety of triggers can launch automated emails, including total cart value, quantity, items in the cart, and more.<!--{% endif %}-->
 
-[User Permissions &amp; Roles]({{ site.baseurl }}{% link system/permissions.md %})
-:  Restrict access to data in the Admin on a “need to know” basis. Create multiple admin roles for read-only or and editing privileges. Track and review all activity at a granular level to specific stores and websites.
+Email Reminders
+:  Send automated [reminder emails]({{ site.baseurl }}{% link marketing/email-reminder-rules.md %}) to customers who have added items to their carts or wish lists, but haven’t made a purchase. A variety of triggers can launch automated emails, including total cart value, quantity, items in the cart, and more.<!--{% endif %}-->
 
-[Full-Page Caching]({{ site.baseurl }}{% link system/cache-full-page.md %})
-:  Enhance performance by caching primary pages. Caching pages improves server response times, reduces load, and increases sustainable traffic. You can use tags to define which components to cache, so only relevant pages are cached as updates take place. It also has the ability to identify and differentiate visitors from shoppers.
+User Permissions and Roles
+:  [Restrict access]({{ site.baseurl }}{% link system/permissions.md %}) to data in the Admin on a “need to know” basis. Create multiple admin roles for read-only or editing privileges. Track and review all activity at a granular level to specific stores and websites.
```

Comment:
cap "Admin roles"?

Reply:
not in this case.... not really a proper noun here